designs/unice/src/elastic.mjs

```diff
@@ -0,0 +1,51 @@
+import { gusset } from './gusset.mjs'
+
+export const elastic = {
```

Comment:
The name of this method will show up in error messages. So I recommend naming it `uniceElastic` so that it's obvious where it comes from when a design is made up of parts pulled in from other designs.

Reply:
As of this pull request, Unice no longer uses parts from any other designs. Should I still change the names?

Reply:
I guess it's good to have unique names regardless, so I've changed them.